daprdocs/content/en/operations/resiliency/configure-policies.md

```diff
@@ -0,0 +1,282 @@
+---
+type: docs
+title: "How-To: Error recovery using resiliency policies"
+linkTitle: "Resiliency Policies"
+weight: 4500
+description: "Configure Dapr error retries, timeouts and circuit breakers"
+---
+
+Resiliency is currently a preview feature. Before you can utilize resiliency policies you must first [enable the resiliency preview feature]({{<ref preview-features >}}).
+
+## Introduction
+
+- TODO: What is resiliency in Dapr?
+- TODO: What problems does it solve?
+
+## Overview
+
+A Dapr resiliency policy allow retry, timeout and circuitbreaker policies to be created and applied to particular targets including specific components and apps (service invocation calls to other apps).
+
+Additionally, resiliency policies can also be [scoped to specific apps]({{<ref "component-scopes.md#application-access-to-components-with-scopes">}}).
+
+In selfhosted mode the resiliency policy must be named `resiliency.yaml` and reside in the components folder provided to the sidecar. In Kubernetes Dapr scans all resiliency policies.
+
+The general structure of a resiliency policy looks like this:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Resiliency
+metadata:
+  name: resiliency
+scopes:
+  # optionally scope the policy to specific apps
+spec:
+  policies:
+    timeouts:
+      # timeout policy definitions
+
+    retries:
+      # retry policy definitions
+
+    circuitBreakers:
+      # circuit breaker policy definitions
+
+  targets:
+    apps:
+      # apps and their applied policies here
+
+    actors:
+      # actor types and their applied policies here
+
+    components:
+      # components and their applied policies here
+```
+
+### Policies
+
+#### Timeouts
+
+Timeouts can be used to early-terminate long-running operations. If a timeout is exceeded the operation in progress will be terminated if possible and an error is returned. Valid values are of the form `15s`, `2m`, `1h30m`, etc
+
+Example definitions:
+```yaml
+spec:
+  policies:
+    # Timeouts are simple named durations.
+    timeouts:
+      general: 5s
+      important: 60s
+      largeResponse: 10s
+```
+
+#### Retries
```

Comment:
What do we mean here when we say "Defaults to X". If I decide to omit that providing a value for that  property, is it still applied implicitly with the default value?